### PR TITLE
Rename Runtime Manager + update related terms

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Renaming the following, starting with the Runtime Manager section to a full Konnect doc cleanup:

* Runtime Manager to Gateway Manager
* Runtime Groups to Control Plane
* Composite Runtime Groups to Control Plane Group
* Runtime Instances to Data Plane Nodes

Based on those main names, we also get:
* "Composite" to "combined" (when referring to config)
* "Runtime", "instance", or "group" to "data plane node"
* "Data plane" to "data plane node"

No screenshots or diagrams in this one as the scope of this PR is already really big.
Will update any needed images separately.

**To do: redirects & verify workflows**

https://konghq.atlassian.net/browse/DOCU-3371

### Testing instructions

https://deploy-preview-6071--kongdocs.netlify.app/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/gateway-manager/data-plane-nodes/renew-certificates.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ you saved earlier:
       --key=/PATH_TO_FILE/tls.key
     ```
 
-1. Open the `values.yaml` file for your runtime instance and update it to point
+1. Open the `values.yaml` file for your data plane node and update it to point
```
Comment (Guaris) [inline app/konnect/gateway-manager/data-plane-nodes/renew-certificates.md:123]:
```suggestion
1. Open the `values.yaml` file for the data plane node and update it to point
```

File: app/konnect/gateway-manager/index.md [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
-* **Team separation:** Dedicate each runtime group to a different team and share
+* **Region separation:** Dedicate each control plane to a region or group of
+regions. Spin up data plane nodes in those regions for each control plane.
+* **Team separation:** Dedicate each control plane to a different team and share
 resources based on team purpose.
 
-![runtime groups](/assets/images/docs/konnect/konnect-runtime-groups-example.png)
-> _**Figure 1:** Example runtime group configuration with three runtime groups: the default group, a development group, and a production group. {{site.konnect_short_name}} is the SaaS-managed global management plane that controls all of the groups, while the runtime groups contain self-managed runtime instances._
+![control planes](/assets/images/docs/konnect/konnect-runtime-groups-example.png)
+> _**Figure 1:** Example control plane configuration with three control planes: the default, a development plane, and a production plane. {{site.konnect_short_name}} is the SaaS-managed global management plane that manages all of the control planes, while the control planes manage configuration for data plane nodes._
```
Comment (Guaris) [inline app/konnect/gateway-manager/index.md:76]:
```suggestion
> _**Figure 1:** Example control plane group configuration for three control planes: the default, a development CP, and a production CP. {{site.konnect_short_name}} is the SaaS-managed global management plane that manages all of the control planes, while the control planes manage configuration for data plane nodes._
```

Comment (Guaris) [reply app/konnect/gateway-manager/index.md:76]: it feels odd to just use the word plane.

File: app/konnect/gateway-manager/index.md [hunk 3]
```diff
@@ -82,81 +90,76 @@ the following {{site.base_gateway}} entities:
...
-### Deleting a runtime group
+### Deleting a control plane
 
 {:.warning}
-> **Warning:** Deleting a group is irreversible. Make sure that you are
-certain that you want to delete the group, and that all entities and runtime
-instances in the have been accounted for.
+> **Warning:** Deleting a control plane is irreversible. Make sure that you are
+certain that you want to delete the control plane, and that all entities and data plane
+nodes in control plane the have been accounted for.
```
Comment (Guaris) [inline app/konnect/gateway-manager/index.md:129]:
```suggestion
nodes in the control plane the have been accounted for.
```

File: app/konnect/gateway-manager/index.md [hunk 4]
```diff
@@ -82,81 +90,76 @@ the following {{site.base_gateway}} entities:
...
-You can also
-choose to manually configure runtime instances on the following:
-* Linux
-* Kubernetes
-* AWS
-* Azure
+The Gateway Manager simplifies data plane node deployment 
+by providing a script to provision a {{site.base_gateway}} data plane node in a 
+Docker container running Linux, on MacOS, or on Windows, eliminating confusion about
+initial configuration or setup.
```
Comment (Guaris) [inline app/konnect/gateway-manager/index.md:159]:
```suggestion
Docker container running Linux, on MacOS, or on Windows. 
```

File: app/konnect/gateway-manager/kic.md [hunk 5]
```diff
@@ -40,19 +40,19 @@ If you don't have an existing KIC deployment, you need the following before usin
...
+> **Note**: The KIC deployment in {{site.konnect_short_name}} is currently read-only. The configuration of the gateway data plane nodes is controlled by changes to resources in the Kubernetes API, and reflected here. For more information about how to manage these resources in KIC, see [Custom Resources](/kubernetes-ingress-controller/latest/concepts/custom-resources/).
 
-![KIC runtime instance dashboard](/assets/images/docs/konnect/konnect-runtime-instance-kic.png)
-> **Figure 1:** This image shows a KIC runtime instance dashboard. For each KIC runtime instance, you can see details about the runtime instance, analytics, and KIC status details.
+![KIC data plane node dashboard](/assets/images/docs/konnect/konnect-runtime-instance-kic.png)
+> **Figure 1:** This image shows a KIC data plane node dashboard. For each KIC data plane node, you can see details about the data plane node, analytics, and KIC status details.
```
Comment (Guaris) [inline app/konnect/gateway-manager/kic.md:49]:
```suggestion
> **Figure 1:** This image shows a KIC data plane node dashboard. For each KIC data plane node, you can see details about an individual data plane node, analytics, and KIC status details.
```

File: app/konnect/network-resiliency.md [hunk 6]
```diff
@@ -4,46 +4,47 @@ toc: false
...
-### What types of data travel between the {{site.konnect_saas}} control plane and the data planes, and how?
+### What types of data travel between the {{site.konnect_saas}} control plane and the data plane nodes, and how?
 
 There are two types of data that travel between the planes: configuration
 and telemetry. Both use the secure TCP port `443`.
 
-* **Configuration:** the control plane sends configuration data to any connected
-  data planes in the cluster.
+* **Configuration:** The control plane sends configuration data to any connected
+  data plane nodes in the cluster.
```
Comment (Guaris) [inline app/konnect/network-resiliency.md:37]:
```suggestion
  data plane node in the cluster.
```

File: app/konnect/network-resiliency.md [hunk 7]
```diff
@@ -4,46 +4,47 @@ toc: false
...
+* **Telemetry:** Data plane nodes send usage information to the control plane
   for Analytics and for account billing. Analytics tracks aggregate traffic by
   service, route, and the consuming application. For billing, Kong tracks the
   number of services, API calls, and active dev portals.
 
 Telemetry data does not include any customer information or any data processed
 by the data plane. All telemetry data is encrypted using mTLS.
 
-### How frequently does data travel between the control plane and data planes?
+### How frequently does data travel between the control plane and data plane nodes?
```
Comment (Guaris) [inline app/konnect/network-resiliency.md:47]:
```suggestion
### How frequently does data travel between the Konnect control plane and data plane nodes?
```

It feels like we have to be specific here. I think this coiuld either be "between the control plane and the data planes", or "....Konnect control plane and the data plane nodes".

**Review by Guaris — CHANGES_REQUESTED** [review]
I don't think "combined" is the right word when referring to config, I think it should remain composite.



"Data plane" to "data plane node" - This can't be a blanket case, and in some cases its not appropriate. 

Data plane node refers to the specific, literal component within a network for lack of a better word, the machine. data plane refers to the broader concept that forwards and proxies traffic. Replacing every instance of data plane with data plane node, risks losing this differentiation. It also risks narrowing the focus of the reader . 

here is an example: 

"The data plane is responsible for efficiently forwarding traffic from the service to the KGWCP"

In this sentence data plane is not referring to the literal component, ie: the docker container, the aws instance, etc. But rather it emphasizes the overfall function of forwarding traffic within the network. 

You replace data plane with data plane node, but then the focus shifts towards the literal component. 
So in this sentence: 

"Each data plane node can be hosted by you on AWS, or even locally."

In this case the sentence is about the component, not about the function. 


This opens us up to write clear and complex sentences like: 

"In the Konnect architecture, the data plane, composed of a combination of self-hosted, and cloud-hosted data plane nodes is responsible for forwarding traffic to the Konnect control plane." 


"Runtime", "instance", or "group" to "data plane node"

This can't be a blanket either, instances and groups  are still a valid words with existing associations outside of Kong that we can use to communicate clearly information inside of Kong.

**Comment by lena-larionova** [comment]
@Guaris I hear you on data plane vs data plane node, but most of the time we are referring to nodes. I'll fix up the places where it wasn't obvious.

Re: Runtime", "instance", or "group" to "data plane node
- was not a blanket change, I went through them one by one. In our docs, we have historically used all three of those terms to refer to what we're now calling "data plane nodes". So, wherever appropriate, I changed them to data plane node or simply node. 
- composite to combined: I think there's only two instances of that term change. I went back and forth on it, wasn't sure if the word "composite" was clear now that there's no composite group. I can change them back.

File: app/konnect/gateway-manager/troubleshoot.md [hunk 8]
```diff
@@ -43,34 +43,34 @@ If you are unable to resolve sync issues using the above methods, contact
...
 control plane is running.
 
-1. Return to the runtime instances page.
+1. Return to the data plane nodes page.
 
-1. Check the runtime instance versions in the table. If you see
-an instance running an older version of {{site.base_gateway}}, your runtime
-instance may need [upgrading](/konnect/runtime-manager/runtime-instances/upgrade/).
+1. Check the data plane node versions in the table. If you see
+a node running an older version of {{site.base_gateway}}, your data plane node
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/konnect/gateway-manager/troubleshoot.md:62]: Why this change? I think it's clear to say "data plane node" in the previous sentence, then just use "node" here. I _don't_ think "data plane" is accurate, as we're talking about individual nodes specifically here.

**Review by Guaris — APPROVED** [review]

**Pull request merged**